app/controllers/accounts_controller.rb

```diff
@@ -15,7 +15,7 @@ def show
   # Does a lookup from the account service in production mode, otherwise returns a stub value
   def lookup
     return AccountService.new.fetch(params[:id]) if Rails.env.production?
-    return {} if params[:id].size < 6
+    return {} unless params[:id].start_with?('jcoyne85')
 
```

Comment:
in development mode, we only have a single fixture faked result, so let's only return it if it looks like it matches (otherwise it can be confusing to developers)